tools/run_onnx.cc

```diff
@@ -414,7 +414,7 @@ class ModelRunner {
     std::vector<std::string> backprop_ins_;
 };
 
-void VerifyOutputs(const InOuts& outputs, const TestCase& test_case, const cmdline::parser& args, bool strict_check) {
+void VerifyOutputs(const InOuts& outputs, const TestCase& test_case, const cmdline::parser& args, bool strict_check, bool show_diff) {
```

Comment:
My bad, I think `strict_check` wasn't a good name. How about renaming this to `check_values`?